mvg/plotting.py

```diff
@@ -222,23 +234,21 @@ def modes_over_time(
 
     ts_range = data["timestamps"].iloc[-1] - data["timestamps"].iloc[0]
     scaling_factor = len(data) / ts_range
+    # List of (y_pos, height) of each row we want to plot
+    row_pos_height = [(0, 0.85 * height), (0.85 * height, 0.2 * height)]
 
-    def _plot_row(row_data, is_uncert_data, y_pos=0):
-        # Collect the indices of data where modes change plus start and end points
-        interval_list = (
-            [0]
-            + [i for i in range(1, len(row_data)) if row_data[i] != row_data[i - 1]]
-            + [len(row_data) - 1]
-        )
-
+    def _plot_row(row_data, is_uncert_data, y_pos=0, row_height=height):
+        # Collect the indices of data where the labels change plus start and end points
+        interval_list = [0] + [
+            i for i in range(1, len(row_data)) if row_data[i] != row_data[i - 1]
+        ]
         for idx in range(len(interval_list) - 1):
             # gray border around uncertains
-            i = interval_list[idx]
+            i, i_next = interval_list[idx : idx + 2]
```

Comment:
I think it is clearer to write this in two lines rather than using a one-liner which uses pythons expansion of lists into multiple assignments and also has the risk of misinterpretation because you need to use `idx+2` as the end in slicing range.
```suggestion
            i = interval_list[idx]
            i_next = interval_list[idx+1]
```